tests/gem5/hello_Ruby/test_ruby.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017 Mark D. Hill and David A. Wood
```

Comment:
Copyright should be Regents of University of California. Check one of the other files we've added recently or there should be this info in darchr/docs, too.